code/_onclick/ai.dm

```diff
@@ -145,17 +126,6 @@
 	else if(hasPower())
 		bolt_drop(usr)
 
-
-/obj/machinery/door/airlock/AIAltClick(mob/living/silicon/ai/user) // Eletrifies doors.
-	if(z != user.z)
-		return
-
-	if(!secondsElectrified)
-		shock_perm(usr)
-	else
-		shock_restore(usr)
-
```

Comment:
Add to changlog and pr body that you remove this

Reply:
yes